scripts/migrations/21

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+# Migration 21 --> 22
+
+set -euo pipefail
+
+# force upgrade multus due to https://github.com/k8snetworkplumbingwg/multus-cni/pull/1445
+kubectl -n cozy-multus get hr multus && {
+	flux -n cozy-multus reconcile hr multus --with-source && \
+	kubectl -n cozy-multus rollout status ds/cozy-multus --timeout=5m
+} || true
```

Comment:
This can be achieved by simple `kubectl` command, like we did it before:

```suggestion
if kubectl get helmrelease multus -n cozy-multus; then
  timestamp=$(date --rfc-3339=ns)
  kubectl annotate helmrelease multus -n cozy-multus \
    reconcile.fluxcd.io/forceAt="$timestamp" \
    reconcile.fluxcd.io/requestedAt="$timestamp" \
    --overwrite
fi
```